contracts/utils/math/Math.sol

```diff
@@ -218,6 +218,62 @@ library Math {
         return result;
     }
 
+    /**
+     * @notice Calculate the modular multiplicative inverse of a number in Z/nZ.
```

Comment:
These don't show up in the docs. I see there are many `@notice`s, so perhaps makes sense to change them all.

```suggestion
     * @dev Calculate the modular multiplicative inverse of a number in Z/nZ.
```